packages/flutter/test/painting/decoration_image_lerp_test.dart

```diff
@@ -178,17 +208,17 @@ void main() {
         return getPixel(x, y);
       }
       const Color lime = Color(0xFF00FF00);
-      expect(getBlockPixel(0), lime); // pure green
-      expect(getBlockPixel(1), lime); // 100% green 0% red
-      expect(getBlockPixel(2), const Color(0xFF19E600));
-      expect(getBlockPixel(3), const Color(0xFF33CC00));
-      expect(getBlockPixel(4), const Color(0xFF808000)); // 50-50 mix green/red
-      expect(getBlockPixel(5), const Color(0xFFCD3200));
-      expect(getBlockPixel(6), const Color(0xFFE61900));
-      expect(getBlockPixel(7), const Color(0xFFFF0000)); // 0% green 100% red
-      expect(getBlockPixel(8), const Color(0xFFFF0000)); // pure red
+      expect(getBlockPixel(0), _matchesColor(lime)); // pure green
+      expect(getBlockPixel(1), _matchesColor(lime)); // 100% green 0% red
+      expect(getBlockPixel(2), _matchesColor(const Color(0xFF1AE600)));
```

Comment:
nit: It's not a huge deal, but reverting this to the original color is a bit cleaner.

Reply:
Done.